clipper_admin/clipper_admin/deployers/caffe2.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import print_function, with_statement, absolute_import
+import shutil
+import torch
+import logging
+import re
+import os
+import json
+
+from ..version import __version__
+from ..clipper_admin import ClipperException
+from .deployer_utils import save_python_function, serialize_object
+
+logger = logging.getLogger(__name__)
+
+
+def create_endpoint(
+        clipper_conn,
+        name,
+        input_type,
+        inputs,
+        func,
+        pytorch_model,
+        default_output="None",
+        version=1,
+        slo_micros=3000000,
+        labels=None,
+        registry=None,
+        base_image="clipper/caffe2-container:{}".format(__version__),
+        num_replicas=1):
+    """Registers an app and deploys the provided predict function with Caffe2 model as
```

Comment:
This function deploys the prediction function with a **PyTorch** model. It serializes the **PyTorch** model in Onnx format and creates a container that loads it as a **Caffe2** model. Let's update the documentation here and for `deploy_caffe2_model` with this information.